src/coreclr/tools/aot/ILCompiler.Compiler/IL/ILImporter.Scanner.cs

```diff
@@ -899,7 +899,7 @@ private void ImportLdToken(int token)
                     {
                         int methodToken = ReadILTokenAt(_currentOffset + 1);
                         var method = (MethodDesc)_methodIL.GetObject(methodToken);
-                        if (IsRuntimeHelpersInitializeArray(method))
+                        if (IsRuntimeHelpersInitializeArrayOrGetSpanData(method))
```

Comment:
Could you update the comment above `// First check if this is a ldtoken Field / InitializeArray sequence.`?